sdk/python/kfp/dsl/io_types_test.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 The Kubeflow Authors
```

Comment:
Add it to existing test file here?
https://github.com/kubeflow/pipelines/blob/master/sdk/python/tests/dsl/io_types_test.py

Reply:
I see. I didn't notice this exists. 
Shouldn't we move that one here? We were trying to adopt the style that unittests files are next to the source files (at least for new tests and sources we're adding).

Reply:
Yeah, moving that here is fine with me as well. Do you mind doing that in this PR?

Reply:
Done. Also moved `metrics_utils_test.py`.